storage/mysql/tree_storage.go

```diff
@@ -34,6 +34,7 @@ const selectLatestSignedRootSql string = `SELECT TreeHeadTimestamp,TreeSize,Root
 		 ORDER BY TreeHeadTimestamp DESC LIMIT 1`
 const insertTreeHeadSql string = `INSERT INTO TreeHead(TreeId,TreeHeadTimestamp,TreeSize,RootHash,TreeRevision,RootSignature)
 		 VALUES(?,?,?,?,?,?)`
+const selectTreeRevisionAtSizeSql string = "SELECT TreeRevision FROM TreeHead WHERE TreeId=? AND TreeSize=?"
```

Comment:
There could be multiple result rows returned by this if nothing is added to the tree between multiple signings, I'd expect they should all have the same `TreeRevision` though.